console/src/components/UIKit/atoms/Tooltip/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import Tooltip from 'react-bootstrap/lib/Tooltip';
+import { StyledTooltip } from './Tooltip';
+
+export type ToolTypeGeneratorProps = {
+  message: string;
+};
+
+export const tooltipGenerator: React.FC<ToolTypeGeneratorProps> = ({
```

Comment:
`tooltipGenerator` was a function from `string` to `JSX.Element`. Could you revert this change?